apps/system/js/bluetooth_transfer.js

```diff
@@ -644,15 +644,17 @@ var BluetoothTransfer = {
     var storage = navigator.getDeviceStorage(storageType);
     var getreq = storage.get(filePath);
```

Comment:
A DOMRequest isn't a promise, so Promise.all will simply create a resolved promise which yields getreq as a parameter. It will work if I use "getreq.then()" to get an actual promise representing the transaction properly however.